### PR TITLE
contrib: test: enable k8s Conformance tests for Fedora

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@jbieren PTAL

**Comment by johnbieren** [comment]
@runcom LGTM.  Let's see what the CI reports. I'm not sure what the kubernetes output looks like.. do we need to do any IP scrubbing before the logs are published publicly to the PR?

**Comment by runcom** [comment]
> @runcom LGTM. Let's see what the CI reports. I'm not sure what the kubernetes output looks like.. do we need to do any IP scrubbing before the logs are published publicly to the PR?

I doubt kube logs reports anything about IPs :confused: but I'll check and hopefully won't miss anything

**Comment by runcom** [comment]
waiting on CI before merging

**Comment by runcom** [comment]
need to fix RHEL failing to start CRI-O at all, wait.

**Comment by runcom** [comment]
retest

**Comment by runcom** [comment]
retest

File: contrib/test/crio-integration-playbook.yaml [hunk 1]
```diff
@@ -253,3 +255,104 @@
   - name: Move all xunit files into one dir to scp
     shell: 'mv /root/src/github.com/kubernetes-incubator/cri-o/test/TestReport-bats*.xml /root/src/github.com/kubernetes-incubator/cri-o/reports/'
     when: xunit
+  # XXX: kube tests from now on
+  - name: git clone k8s repo
+    git:
+      repo: https://github.com/kubernetes/kubernetes
+      dest: /root/src/k8s.io/kubernetes
+      version: 2899f47bc
```
Comment (mrunalp) [inline contrib/test/crio-integration-playbook.yaml]: I think in the future we should pick this from lock.json.

**Comment by runcom** [comment]
> 51/51 passed on Fedora - Passed.
> 120/121 k8s node-e2e passed on Fedora - Failed.

:tada:

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**